script/node-spec-runner.js

```diff
@@ -0,0 +1,35 @@
+const cp = require('child_process')
+const fs = require('fs')
+const path = require('path')
+
+const BASE = path.resolve(__dirname, '../..')
+const NODE_DIR = path.resolve(BASE, 'third_party', 'electron_node')
+const NPX_CMD = process.platform === 'win32' ? 'npx.cmd' : 'npx'
+
+const utils = require('./lib/utils')
+const { YARN_VERSION } = require('./yarn')
+
+if (!process.mainModule) {
+  throw new Error('Must call the node spec runner directly')
```

Comment:
why check this?

Reply:
I copied from `nan-spec-runner.js`.  Don't want folks randomly requir'ing this file by accident (it'll just run the tests on-require)